package.json

```diff
@@ -41,5 +41,6 @@
     "vue-router": "3.6.5",
     "vuelidate": "0.7.6",
     "vuex": "3.6.0"
-  }
+  },
+  "contractsVersion": "2.0.3"
```

Comment:
FYI, this shouldn't be modified here (but in say, the TodoMVC app).

The `chel pin` command should check to see if `package.json` exists in the current folder, and if it does, it should modify the `"contractsVersion"`.

And if `package.json` doesn't exist, then it can create/modify a file call `chelonia.json`.